### PR TITLE
3/17

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: emails/2023-03-17.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+In this mode, you set up Dolt as a replica of your primary MySQL database. Every write to the primary is replicated to Dolt and a new Dolt commit is created. This means you get the ability to find bad changes to your primary MySQL and create SQL patches to roll bad changes back. [The blog](https://www.dolthub.com/blog/2023-03-15-getting-started-versioned-mysql-replica/) has detailed example. You can do all this without migrating your primary off of MySQL. Just add Dolt.
+
+### Dolt 0.75
+
+Just in case you missed last week, we turned the chunk journal on by default making [the current Dolt release](https://github.com/dolthub/dolt/releases/tag/v0.75.3) ACID compliant. Upgrade now. Beyond durability it is also faster. We are down to [1.9X MySQL on sysbench benchmarks](https://docs.dolthub.com/sql-reference/benchmarks/latency).
+
+The next step is Dolt 1.0 which you should see in the next quarter. It's been a long slog but Dolt is stable and ready for all your production workloads. If you've been waiting, now is the time to make the switch.
```
Comment (tbantle22) [inline emails/2023-03-17.md:21]: This could be a hosted plug

**Review by tbantle22 — APPROVED** [review]

File: emails/2023-03-17.md (hunk 1 above)
Comment (timsehn) [reply emails/2023-03-17.md]: Good call.

File: emails/2023-03-17.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+[![Versioned MySQL Replica](../images/versioned-mysql-replica.png)](https://www.dolthub.com/blog/2023-03-15-getting-started-versioned-mysql-replica/)
+
+In this mode, you set up Dolt as a replica of your primary MySQL database. Every write to the primary is replicated to Dolt and a new Dolt commit is created. This means you get the ability to find bad changes to your primary MySQL and create SQL patches to roll bad changes back. [The blog](https://www.dolthub.com/blog/2023-03-15-getting-started-versioned-mysql-replica/) has detailed example. You can do all this without migrating your primary off of MySQL. Just add Dolt.
+
+### Dolt 0.75
+
+Just in case you missed last week, we turned the chunk journal on by default making [the current Dolt release](https://github.com/dolthub/dolt/releases/tag/v0.75.3) ACID compliant. Upgrade now. Beyond durability it is also faster. We are down to [1.9X MySQL on sysbench benchmarks](https://docs.dolthub.com/sql-reference/benchmarks/latency).
```
Comment (max-hoffman) [inline emails/2023-03-17.md:19]: Should we start saying we are 90% slower than MySQL? Or wait until we are more like <50%.

Comment (timsehn) [reply emails/2023-03-17.md:19]: I've had the same thought...

I think 90% slower sounds worse than 2X

**Pull request merged**